packages/launchpad/cypress/e2e/support/getPathForPlatform.ts

```diff
@@ -0,0 +1,11 @@
+import { decodeBase64Unicode } from '@packages/frontend-shared/src/utils/base64'
+
+export function getPathForPlatform (posixPath: string) {
+  // @ts-ignore
+  const cy = window.Cypress
+  const platform = cy?.platform || JSON.parse(decodeBase64Unicode(window.__CYPRESS_CONFIG__.base64Config)).platform
```

Comment:
This code is really ugly and seems like it shouldn’t be necessary. Also isn’t it Cypress.platform not cy.platform? 

Also what’s the situation where this hasn’t been decided yet?

Reply:
I moved this code to reuse it, I'm not sure what prompted the original structure of it.

Reply:
Fixed here: [`dcc4122` (#21354)](https://github.com/cypress-io/cypress/pull/21354/commits/dcc4122f12b1e3aef14bd72acfd79b47af73203b)

(This is used for validation in tests by the way, not in application code)